test/dns_dnscmd/dnscmd_test.rb

```diff
@@ -153,4 +166,18 @@ def test_dnscmd_enum_zones_parses_primary_zones_only
       "domain.local",
       "TrustAnchors"], Proxy::Dns::Dnscmd::Record.new.enum_zones
   end
+
+  def test_enum_records
```

Comment:
Sure.  Actually I think the more important test is when there are _no_ PTR records returned because the DNS server automatically deleted them when the A record was removed.  I'll add both.